open-craft/ansible-playbooks
# chore: upgrade mysql version used for jira to 5.7

## Description

This PR contains changes to upgrade the MySQL version to 5.7.

The Ansible Role `geerlingguy.mysql` points back to the upstream repository as I couldn't
find any diverging changes made in the OpenCraft repo. In addition, the old version
didn't work on Ubuntu 20.40 due to still relying on Python2 libraries.

## Supporting information

- https://tasks.opencraft.com/browse/SE-5555

## Testing instructions

```bash
ansible-playbook -vvv deploy/playbooks/tasks.yml -l new-mysql.tasks.opencraft.com --private-key=~/.ssh/vm.pem
```

Will deploy the changes, but it should be enough to SSH onto `new-mysql.tasks.opencraft.com` and verify that

- MySQL is running and on v5.7
- `/dev/vdb` is mounted to `/var/lib/mysql/`

## Deadline

30 May 2022
